youtube_dl/extractor/jove.py

```diff
@@ -58,16 +50,16 @@ def _real_extract(self, url):
         if not video_url:
             raise ExtractorError('Failed to get the video URL')
 
+        if "mp4" not in video_url:
+            raise ExtractorError('This video is DRM protected.', expected=True)
+
         title = self._html_search_meta('citation_title', webpage, 'title')
         thumbnail = self._og_search_thumbnail(webpage)
         description = self._html_search_regex(
```

Comment:
Valid HTML could include whitespace or other attributes in the tag, or use '' instead of "", something like this:
```
        description = self._html_search_regex(
            r'<p\s[^>]*?class\s*=\s*(?P<q>["|\'])jove_content(?P=q)[^>]*>(?P<desc>[^<]+)</p>',
            webpage, 'description', fatal=False, group='desc')
```
In case of valid but deprecated HTML with unquoted attribute values, `(?P<q>"|\'|\b)` would match the start of the class attribute value.